### ISSUE TITLE
webglFillColor incorrect colouring

### ISSUE THREAD

**Issue by OliverForeman** [d3fc/d3fc#1466]
When using a decorate function to color a bar series we came across an issue where the divisor appears to be ignored. This results in the colours being used as if they were per vertex (divisor of `0`) instead of per element (divisor of `1`).

Decorate function that displayed a problem:
```javascript
.decorate(function(program, data) {
        const webglColor = (_, i) => {
            const rgba = d3.color(color(i));
            return [rgba.r / 255, rgba.g / 255, rgba.b / 255, rgba.opacity];
        };

        fc.webglFillColor(webglColor).data(data)(program);
    });
```

Changing the divisor from `1` to another value removes the issue and the bars are all drawn with the expected colours. This seems to indicate an issue specifically related to the divisor being `1`.

Changing the attribute passed to the shader to a `vec2`/`vec3` removes the issue, this seems to imply some relationship between the `size` property and the `divisor` property that is causing the problem.

Removing an attribute from the shader also seems to fix the issue implying a size/memory limit might have been reached for passing in data to the shaders.

Changing the order that the attributes are declared in within the vertex shader apears to fix the problem. This seems to be related to the location of the vertex attribute, i.e. the location from `gl.getAttribLocation()`.

After checking out the example on different machines it seems this is likely a hardware issue as the problem does not exist on any other machine. A restart/windows update of the computer showing the issue seemed to fix the problem ([windows update KB4465065](https://support.microsoft.com/en-gb/help/4465065/kb4465065-intel-microcode-updates)).

**Comment by OliverForeman** [comment]
After encountering this issue again the problem was resolved by restarting the computer.

**Comment by chrisprice** [comment]
Next time we see this issue please can we check if restarting Chrome fixes the problem. We need to get to the bottom of whether this is a graphics driver issue or a Chrome issue.

**Comment by chrisprice** [comment]
Fixed by a53813f561644e1f57e9e4340655072160282a54
